spec/services/container_dashboard_service_spec.rb

```diff
@@ -1,4 +1,5 @@
 require "spec_helper"
+require 'recursive-open-struct'
```

Comment:
> We should probably add recursive-open-struct to the Gemfile. Right now it works only because kubeclient and openshift-client depend on it.

I wasnt aware of this and I can see that it ```require 'recursive-open-struct'``` also appears in another file. Ill open a separate PR for this.